src/test/java/uk/gov/justice/hmpps/probationteams/repository/LocalDeliveryUnitRepositoryTest.kt

```diff
@@ -17,8 +17,8 @@ import uk.gov.justice.hmpps.probationteams.utils.uniqueLduCode
 import java.util.UUID
 
 @ActiveProfiles("test")
-@ExtendWith(SpringExtension::class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@EnableAutoConfiguration(exclude = [SpringDocKotlinConfiguration::class])
```

Comment:
this was causing an issue in latest springdoc as required beans were not included when web env is NONE